.github/workflows/ci.yml

```diff
@@ -8,8 +8,8 @@ on:
   workflow_dispatch:
 
 jobs:
-  nikola-build:
-    uses: ./.github/workflows/reusable-nikola-build.yml
```

Comment:
If this file is changed to do container builds, how with the RTD previews get built? It looks like this file is currently run on both `pull_request` and `push: branches: "main"` Am I missing something else for that?

Reply:
There is both a CI check that builds the website with nikola and the RTD preview build, which is configured on the server-side.

It might be a little more than necessary but it ensures that every PR is built in a consistent way, i.e. on the same stack, and we're not actually relying on RTD to verify the build.

Reply:
for completeness, we've moved the CI debate to a future PR